inst/pages/taxonomy.qmd

```diff
@@ -68,14 +68,14 @@ columns that `mia` assumes to contain the taxonomic information.
 taxonomyRanks(tse)
 ```
 
-This can then be used to subset the `rowData` to columns needed.
+This can then be used to subset the `rowData` to focus only on the taxonomy-related columns.
 
 ```{r}
 rowData(tse)[, taxonomyRanks(tse)]
 ```
 
 `taxonomyRankEmpty()` checks for empty values in the given `rank` and returns a 
-logical vector of `length(x)`.
+logical vector of `length(x)`  (where `length(x)==ncol/rowData(tse)[, taxonomyRanks(tse)]`).
```

Comment:
taxonomyRankEmpty() returns TRUE/FALSE for each row. It checks if the row does not have any taxonomy information on level `rank` --> length is nrow(tse)